### PR TITLE
added responsive and hover effects

### PR DESCRIPTION
**Pull request by Towko**

### CODE REVIEW

File: src/pages/Main/components/Technologies/Technologies.module.scss [hunk 1]
```diff
@@ -13,14 +29,47 @@
...
+                    right: 0;
+                    transition: .5s;
+                }
+            }              
+        }
+        &:hover {
+            .technoPic__title {
+                right: 0;
+                transition: .5s;
+            }
```
Comment (ostannja) [inline src/pages/Main/components/Technologies/Technologies.module.scss:53]: please add hover to .technoPic__title for screen resolution under 768px

File: src/pages/Main/components/Technologies/Technologies.module.scss [hunk 2]
```diff
@@ -13,14 +29,47 @@
...
-    &__ContBig {
+    &__title {
         display: flex;
-        position: relative;
-        width: 49.523%;
-        max-height: 24.0625rem;
-        margin: 0.375rem 0;
+        position: absolute;
+        width: 17.25rem;
+        height: 3.875rem;
```
Comment (ostannja) [inline src/pages/Main/components/Technologies/Technologies.module.scss:61]: is too little (espesially in width) for the screen resolution less then 768px, relatively to parent element (must be 225px in resolution 375px, and now it is 130px)

File: src/pages/Main/components/Technologies/Technologies.module.scss [hunk 3]
```diff
@@ -32,4 +81,21 @@
...
+                width: 100%;
+                max-height: 145px;
+            }
+        }
+        &__title {
+            right: unset;
+            left: 0;
+        }
+    }
 }
```
Comment (ostannja) [inline src/pages/Main/components/Technologies/Technologies.module.scss]: add empty line

File: src/pages/Main/components/Technologies/Technologies.module.scss (hunk 1 above)
Comment (Towko) [reply src/pages/Main/components/Technologies/Technologies.module.scss:53]: What do you mean? Why? It's everywere is that is need

**Pull request merged**